Microsoft.Toolkit.Uwp.UI.Controls.Layout/GridSplitter/SplitBase.cs

```diff
@@ -0,0 +1,220 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Base class for splitting/resizing controls
+    /// </summary>
+    public partial class SplitBase : Control
+    {
+        /// <summary>
+        /// Vertical symbol for GripperBar in Segoe MDL2 Font asset.
+        /// </summary>
+        protected const string GripperBarVertical = "\xE784";
+
+        /// <summary>
+        /// Horizontal symbol for GripperBar in Segoe MDL2 Font asset.
+        /// </summary>
+        protected const string GripperBarHorizontal = "\xE76F";
+
+        /// <summary>
+        /// Distance (horizontal or vertical) to move, in response to keyboard activity.
+        /// </summary>
+        protected const double GripperKeyboardChange = 8.0d;
+
+        /// <summary>
+        /// Font family used for gripper.
+        /// </summary>
+        protected const string GripperDisplayFont = "Segoe MDL2 Assets";
```

Comment:
So this is duplicated between the XAML and the C#? Wonder if we should define a resource key in the XAML instead?